.travis.yml

```diff
@@ -113,3 +113,4 @@ script:
     # Prepare the Electron app and build it
     cd ElectronClient
     USE_HARD_LINKS=false npm run dist
+    bash generate_sha256.sh
```

Comment:
As I understand, `npm run dist` builds the app then upload the packages to GitHub. So since your hash generation happens after, I guess it won't be uploaded to GitHub, or am I missing something?

Reply:
You are right. This won't work.
I am trying to rework this using Electron Builder hooks.